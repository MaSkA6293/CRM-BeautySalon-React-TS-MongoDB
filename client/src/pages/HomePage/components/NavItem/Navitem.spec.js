import React from "react";
import Navitem from "./index";

const setUp = (props) => shallow(<Navitem {...props} />);

describe("all tests Navitem", () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });
  describe("Navitem component", () => {
    it("should render Navitem component", () => {
      expect(component).toMatchSnapshot();
    });
  });
});
