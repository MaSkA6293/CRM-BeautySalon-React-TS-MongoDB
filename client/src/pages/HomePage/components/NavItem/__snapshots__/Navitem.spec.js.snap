// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`all tests Navitem Navitem component should render Navitem component 1`] = `
<div
  className="nav-item"
>
  <div
    className="nav-item__icon"
  >
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon={null}
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size={null}
      spin={false}
      swapOpacity={false}
      symbol={false}
      title=""
      transform={null}
    />
  </div>
  <span
    className="nav-item__title"
  />
</div>
`;
